dopamine/discrete_domains/run_experiment/__init__.py

```diff
@@ -0,0 +1,24 @@
+# coding=utf-8
+# Copyright 2019 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Module defining classes and helper methods for general agents."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+# pylint: disable=wildcard-import
+from dopamine.discrete_domains.run_experiment.helper import *
```

Comment:
I think I understand the point of this as making your `run_experiment` refactor backwards compatible.  I guess ideally we wouldn't have to worry about that backwards compatibility.  But if we do, why not just keep the same old `run_experiment.py` and put new stuff in the new `run_experiment` subpackage?

Reply:
the point is more to avoid circular import than backward compatibility.

As suggested I moved everything to the same file.